appium/tests/screenobjects/base.screen.ts

```diff
@@ -40,19 +33,9 @@ export default class BaseScreen {
   }
 
   static checkErrorModal = async (errorText: string) => {
-    const message = '-ios class chain:**/XCUIElementTypeAlert/XCUIElementTypeOther/XCUIElementTypeOther/' +
-      'XCUIElementTypeOther[2]/XCUIElementTypeScrollView[1]/XCUIElementTypeOther[1]/XCUIElementTypeStaticText[2]';//it works only with this selector
-    await expect(await this.errorHeader).toBeDisplayed();
-    expect(await (await $(message)).getAttribute('value')).toContain(errorText);
-    await expect(await this.okButton).toBeDisplayed();
-  }
-
-  static checkErrorModalForFES = async (errorText: string) => {
-    const message = '-ios class chain:**/XCUIElementTypeAlert/XCUIElementTypeOther/XCUIElementTypeOther/' +
-        'XCUIElementTypeOther[2]/XCUIElementTypeScrollView[1]/XCUIElementTypeOther[1]/XCUIElementTypeStaticText[2]';//it works only with this selector
-    await expect(await this.errorFESHeader).toBeDisplayed();
-    expect(await (await $(message)).getAttribute('value')).toContain(errorText);
-    await expect(await this.retryButton).toBeDisplayed();
+    await expect(await this.currentModal).toBeDisplayed();
```

Comment:
Here better naming will be `checkModalMessage` with `message` parameter, as we'll check not only errors, but other messages too.

Reply:
ok, updated